src/ui/components/atoms/Author-Label/styles.scss

```diff
@@ -0,0 +1,29 @@
+@import '../../../../globals/breakpoints';
+@import '../../../../globals/mixins';
+@import '../../../../globals/variables';
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@200;275;300&display=swap');
```

Comment:
We already have these fonts embedded in the application use the embed fonts directly don't use google fonts